.circleci/config.yml

```
@@ -395,7 +395,7 @@ jobs:
             mkdir -p /tmp/${DATASET}/derivatives
             sudo setfacl -d -m group:$(id -gn):rwx /tmp/${DATASET}/derivatives
             sudo setfacl -m group:$(id -gn):rwx /tmp/${DATASET}/derivatives
-            pip install --upgrade pip setuptools
+            pip install --upgrade "pip<21" "setuptools<45"
             pip install --upgrade /tmp/src/fmriprep/wrapper/
```

Can we assume that these images have Python 3 as well? If so, I think probably the cleaner long-term solution is this:

```suggestion
            python3 -m pip install --upgrade pip setuptools
            python3 -m pip install --upgrade /tmp/src/fmriprep/wrapper/
```

Keeping the Py2 accommodations to be both explicit and minimal is preferable.